src/main/groovy/com/ullink/util/GradleHelper.groovy

```diff
@@ -0,0 +1,15 @@
+package com.ullink.util
+
+import org.gradle.api.Project
+import org.gradle.api.UnknownTaskException
+
+class GradleHelper {
+    def static String getPropertyFromTask(Project project, String property, String task) {
+        try {
+            def theTask = project.tasks.getByName(task)
```

Comment:
`findByName` will return null if not found, better than using exceptions as it's a pretty valid use-case